src/actions/flowLimitation.js

```diff
@@ -21,7 +21,7 @@ export const handleFlowLimitation = () => (dispatch, getState) => {
             accountBalance: true
         }));
     } 
-    else if (redirectUrl.includes(WALLET_SIGN_URL)) {
+    else if (redirectUrl === `/${WALLET_SIGN_URL}`) {
```

Comment:
Tested and works well. However it seems that if I exit the `/sign` flow before clicking 'allow', then navigate to `/sign-in-ledger` or `/recover-seed-phrase` or `/recover-account` and refresh the page - the `url` object gets re-populated and the nav links disappear.

Also, could `redirectUrl.includes(WALLET_LOGIN_URL)` on `L15` have the same issue with the `cli-login-success` route? Also I wonder why login URLs are always `/login/` with the extra `/`, but sign URLs are `/sign`?